README.md

```diff
@@ -1 +1,5 @@
-# korean-wordle
\ No newline at end of file
+# Guess the word in Korean!
+by Emilija
+
+## Project description
+Enter Korean words until you guess the answer
```

Comment:
That is a pretty cool idea. How would the inputs look like or rather how would you display the interface?

Reply:
Still working on the interface details but users will probably click on character options. Will add the missing parts too! Thanks for the feedback